packages/core/content-type-builder/server/src/services/builder.ts

```diff
@@ -1,18 +1,31 @@
+// NOTE: All these reserved names MUST be in camelCase, because they will also be converted to snake_case and compared against incoming values
```

Comment:
Not sure I understand this statement since none of the reserved names below is written in camelCase

Reply:
oops, comments got out of sync as I updated, removing this one in favor of the way it's written and implemented